mxcube3/__init__.py

```diff
@@ -110,6 +111,7 @@ def complete_initialization(app):
         app.empty_queue = pickle.dumps(hwr.getHardwareObject(cmdline_options.queue_model))
         app.sample_changer = app.beamline.getObjectByRole("sample_changer")
         app.rest_lims = app.beamline.getObjectByRole("lims_rest_client")
+        app.queue = qutils.new_queue()
```

Comment:
Hm I agree with you that it seems strange. The app.empty_queue is a serialized version of the empty queue that is used to initialize a new queue, Its perhaps better to just instantiate a QueueModel Object directly. Or do you have another suggestion ? I thought it was done in that why for a reason so I didn't question it to much.